geode-pulse/src/main/java/org/apache/geode/tools/pulse/internal/service/ClusterDetailsService.java

```diff
@@ -44,14 +45,20 @@
 public class ClusterDetailsService implements PulseService {
 
   private final ObjectMapper mapper = new ObjectMapper();
+  private final Repository repository;
+
+  @Autowired
```

Comment:
no need for this constructor if Repository variable is declared as `autowired` already.

Same should be the same for all the service classes.